packages/p2p/src/components/advertiser-page/advertiser-page-dropdown-menu.jsx

```diff
@@ -35,7 +34,7 @@ const AdvertiserPageDropdownMenu = ({ is_my_advert }) => {
                         onClick={advertiser_page_store.showBlockUserModal}
                     >
                         <Dropdown
-                            className={`advertiser-page__dropdown-container${is_my_advert ? '--disabled' : ''}`}
+                            className='advertiser-page__dropdown-container'
                             is_align_text_right
```

Comment:
Clarified with design team, there are currently no cases where the dropdown is disabled, and thus the prop `is_my_advert` is no longer necessary in this case